### ISSUE TITLE
Topic: Manage co-creators box shows all participants

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#1010]
Something needing review by @kevincrepin at some point, this manage co-creators box currently shows all participants (not just admins and editors). Should it? And what happens when there are say 50 or 100, currently looks like it will be one very long list down the side of the edit flow :) 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/a7b6c5d5-1a30-4059-9d2b-483e280e327f)

**Comment by BeccaMelhuish** [comment]
Will link to [this issue](https://github.com/citizenos/citizenos-fe/issues/1007) for @kevincrepin's reference when reviewing, as is connected :)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish I removed the users with participant permissions from the co-editors list. Though it will probably need API side update to exclude them from results and this way possibly add pagination for long list of editors

**Comment by BeccaMelhuish** [comment]
Ok yes, though I do think this bit needs some review from @kevincrepin still anyway - to check the UX is right, don't want to rush to make changes that don't make sense for the users' journey(s) :)

**Comment by ilmartyrk** [comment]
Add a filter to get only users that have edit or higher permissions level, add scrollbar/pagination if the list gets too long

**Comment by kevincrepin** [comment]
I agree that it probably shouldn't scale into infinitum, but let's also not add a scrollbar into this. I suggest we do the good old "+ x more" which would open up the popup form where to manage:

![Property 1=Variant4](https://github.com/user-attachments/assets/ac23504c-4473-4aaa-981c-73f62b364b84)

@ilmartyrk you can find this component variant here: https://www.figma.com/design/gOJ4i00538e0qaP6IKIPVE/Platform-2.0---Design?node-id=137-28657&t=3hHy3U6W9ohEF2Lu-1

**Comment by BeccaMelhuish** [comment]
**Dev task summary:**

- [ ] Make sure only those with admin or editor rights show up in the 'Co-creators' box in topic edit view. 
- [ ] Make the design as per Kevin's design above
